pennylane/fermi/conversion.py

```diff
@@ -126,3 +130,86 @@ def _(fermi_operator: FermiSentence, ps=False, wire_map=None):
         return qubit_operator.map_wires(wire_map)
 
     return qubit_operator
+
+
+@jordan_wigner.register
+def _jordan_wigner_legacy(op: list, notation="physicist"):  # pylint:disable=too-many-branches
```

Comment:
Is anything in this function changed, except its name?

Reply:
Name and location only, I copy-pasted from the `observable_hf` file to here and updated the name